### ISSUE TITLE
Kiali multicluster crashes when one cluster is offline

### ISSUE THREAD

**Issue by lucasmellos** [kiali/kiali#5122]
Env:
2 K8S cluster
East-west gateways 
Latest kiali operator

Description:
Earlier this week I had an issue in one of my clusters that led K8S control plane to exhaustion (nothing related to istio/kiali) when I attempted to login onto kiali health cluster I was prompted to that error when some componentes aren't healthy. 
The logs have shown that Kiali west wasn't able to reach Kiali on east. 
Is it a prerequisite? Any workaround in cases like these?

**Comment by israel-hdez** [comment]
@lucasmellos Can you, by chance, collect the logs of the Kiali at the time of the crash?
I would be interested to see where it crashes.

**Comment by lucasmellos** [comment]
Unfortunately I don't have them anymore, @israel-hdez. But to help you understand it more deeply. My istio's have External CA by Hashicorp Vault.

**Comment by lucasmellos** [comment]
Hi @israel-hdez have you setup any "eastwest" gateway? In my scenario my clusters have used external CA for authentication and they communicate using ingress gateways

**Comment by israel-hdez** [comment]
> Hi @israel-hdez have you setup any "eastwest" gateway? In my scenario my clusters have used external CA for authentication and they communicate using ingress gateways

Yes, I have the eastwest gateway. It is, anyway, not relevant to Kiali, because Kiali does not use the gateways for its communications. The gateways are scoped to traffic of mesh services and, usually, Kiali does not belong to the mesh.

I still would like to know what are your Istio and Kiali versions to try with your versions. I do remember that in the past Kiali did crash but that was fixed several releases ago. So, I want to be sure it is not the case.

**Comment by lucasmellos** [comment]
Hi @israel-hdez. They're
```
quay.io/kiali/kiali-operator:v1.51.0
quay.io/kiali/kiali:v1.51.0
gcr.io/istio-release/pilot:1.13.4
```

**Comment by jshaughn** [comment]
I don't think this is something we can replicate and we'll have to ask for outside help.

**Comment by jshaughn** [comment]
Closing as stale, and likely obsolete given major changes in Kiali multi-cluster support impl.
